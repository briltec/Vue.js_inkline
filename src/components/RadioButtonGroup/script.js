import IInputGroup from 'inkline/components/InputGroup';
import IButtonGroup from 'inkline/components/ButtonGroup';

export default {
    name: 'IRadioButtonGroup',
    extends: IInputGroup,
    components: {
        IButtonGroup
    },
    created () {
        this.$on('change', value => {
            this.onInput(value);
        });
    }
};
