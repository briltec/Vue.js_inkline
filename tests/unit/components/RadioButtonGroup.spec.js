import { mount } from '@vue/test-utils';
import RadioButtonGroup from 'inkline/components/RadioButtonGroup';

describe('Components', () => {
    describe('RadioButtonGroup', () => {
        let wrapper;

        beforeEach(() => {
            wrapper = mount(RadioButtonGroup);
        });

        it('should render correctly', () => {
            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
