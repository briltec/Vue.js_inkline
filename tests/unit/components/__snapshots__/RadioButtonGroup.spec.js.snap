// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components RadioButtonGroup should render correctly 1`] = `<div role="radiogroup" class="button-group form-group -radio"></div>`;
